opencog/reasoning/RuleEngine/rule-engine-src/pln/PLNCommons.cc

```diff
@@ -84,39 +83,34 @@ Handle PLNCommons::create_bindLink(
     return as_->addLink(BIND_LINK, var_listLink, implicationLink);
 }
 
-HandleSeq PLNCommons::get_nodes(const Handle& hinput,
-                                vector<Type> required_nodes)
+UnorderedHandleSet PLNCommons::get_nodes(const Handle& hinput,
```

Comment:
FYI, there is a perhaps simpler, easier-to-use version of this method in the FindVars class in /opencog/query/PatternUtils.h  -- there are a bunch of this kind of utilities there.